infra/main.parameters.json

```
@@ -110,6 +113,9 @@
     "embeddingDeploymentVersion":{
       "value": "${AZURE_OPENAI_EMB_DEPLOYMENT_VERSION}"
     },
+    "cembeddingDeploymentSkuName":{
+      "value": "${AZURE_OPENAI_EMB_DEPLOYMENT_SKUNAME}"
```

Ditto, for similarity with other SKU variables:

```suggestion
      "value": "${AZURE_OPENAI_EMB_DEPLOYMENT_SKU}"
```